src/auth/auth.service.ts

```diff
@@ -1,42 +1,15 @@
 import { Injectable, UnauthorizedException } from '@nestjs/common';
-import { JwtService } from '@nestjs/jwt';
 import * as bcrypt from 'bcrypt';
 import { UserService } from '../user/user.service';
 import { User } from '../db/models/User.model';
-import { classToPlain } from 'class-transformer';
 
 @Injectable()
 export class AuthService {
-  constructor(
-    private readonly userService: UserService,
-    private jwtService: JwtService,
-  ) {}
+  constructor(private readonly userService: UserService) {}
 
-  async validateUser(email: string, password: string): Promise<User | null> {
+  async validateUser(email: string): Promise<User | null> {
     const user = await this.userService.getUserById(email);
-    if (!user) return null;
-    const passwordValid = await bcrypt.compare(password, user.password);
-    if (!user || !passwordValid) {
-      throw new UnauthorizedException(
-        'Could not find the user/password combination',
-      );
-    }
-    if (user && passwordValid) return user;
+    if (user) return user;
```

Comment:
Actually going to remove this file in a later PR since all this is handled by cognito now. This is dead code